eclipse-collections-code-generator/src/main/resources/api/lazyPrimitiveIterable.stg

```diff
@@ -52,6 +52,12 @@ public interface Lazy<name>Iterable extends <name>Iterable
      */
     \<V> LazyIterable\<V> flatCollect(<name>ToObjectFunction\<? extends Iterable\<V>\> function);
 
+    @Override
+    default RichIterable\<PrimitiveIterable> chunk(int size)
+    {
+        throw new UnsupportedOperationException(this.getClass().getSimpleName() + ".chunk() not implemented yet");
```

Comment:
I see a lot of UnsupportedOperationExceptions. Is this a temporary thing for the 9.0 release?

Reply:
I think the UnsupportedOperationExceptions are ok to throw in the default implementations so this can now be introduced in a minor release like 9.1.  The implementation classes should provide overrides with the appropriate implementations.